matrix/common/pandas_utils.py

```diff
@@ -0,0 +1,41 @@
+import pandas
+from pandas.core.frame import DataFrame
+from zarr import Group
+
+
+def convert_dss_zarr_root_to_subset_pandas_dfs(zarr_root: Group, start_row: int, end_row: int):
+    """
+    Params:
+        zarr_root: instance of Zarr.group
+        start_row: beginning row index to filter
+        end_row: end row index to filter
+    Returns:
+        Tuple of two pandas.DataFrames:
+            expression_df: subset filtered expression counts
+            qc_df: subset filtered qc values
+    """
+    expression_df = pandas.DataFrame(data=zarr_root.expression[start_row:end_row],
+                                     index=zarr_root.cell_id[start_row:end_row],
+                                     columns=zarr_root.gene_id)
+    qc_df = pandas.DataFrame(data=zarr_root.cell_metadata[start_row:end_row],
+                             index=zarr_root.cell_id[start_row:end_row],
+                             columns=zarr_root.cell_metadata_name[start_row:end_row])
+    return expression_df, qc_df
+
+
+def pass_filter_through_pandas_dfs(filter_query: str, exp_df: DataFrame, qc_df: DataFrame):
```

Comment:
Thoughts on renaming to `apply_filter`? And if above fn returns a Tuple, this could accept a Tuple too